[improvement] Built error message "Locale ID 'names:${id}' is missing in the calendar '${calendar.name}'"

Build process (npm run build)  show me the message "Error: Locale ID 'names:dominic_of_the_mother_of_god_barberi_priest' is missing in the calendar 'England'."  (this was an intentional error), I look on "England" calendar  for the mistake, and nothing.
Take a look on '/rites/roman1969/build/bundle.ts' to understand what is happening.
This message is  inside a IF statement.

>         if (locale.names && Object.prototype.hasOwnProperty.call(locale.names, id)) {
>             update[id] = locale.names[id];
>           } else if (locales.En.names && Object.prototype.hasOwnProperty.call(locales.En.names, id)) {
>             update[id] = locales.En.names[id];
>           } else {
>             throw new Error(`Locale ID 'names:${id}' is missing in the calendar '${calendar.name}'.`);
>           }

The message tests LOCALES EN, but refers to a locale where is used.
Locale EN should be have ALL 'names' , it is OK, but the message did not show it. 
SUGGESTIONs to line 165: 
1) throw new Error(`Locale ID 'names:${id}' is missing in calendar locale en.ts.`);
2) throw new Error(`Locale ID 'names:${id}' SHOULD BE present in the calendar locale en.ts.`);

Shoud to refers to locale/en.ts file

To reproduce, just comment a name in **/rites/roman1969/src/locales/en.ts** e run a 'npm run build'
To show this error I comment Saint Claire in locale/en.ts:
```
    christopher_of_palestine_martyr: 'Saint Christopher, Martyr',
    ciaran_of_clonmacnoise_abbot: 'Saint Ciaran, Abbot',
    clair_of_normandy_priest: 'Saint Clair of Normandy, Priest and Martyr',
    //clare_of_assisi_virgin: 'Saint Clare, Virgin',    //<== I PUT THIS COMMENT
    clement_i_pope: 'Saint Clement I, Pope and Martyr',
    clement_mary_hofbauer_priest: 'Saint Clement Mary Hofbauer, Priest',
    clement_of_ohrid_and_gorazd_of_moravia_bishops_and_companions:
      'Saints Clement of Ohrid and Gorazd, Bishops, and Companions',
```

run npm run build, this is the error:
```
✓ Generate calendar bundle files into ./tmp/bundles/
[░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░] 0% | ETA: 0s | 0/870 | {filename}/home/viajante/projects/romcal-tst/rites/roman1969/build/bundle.ts:165
            throw new Error(`Locale ID 'names:${id}' is missing in the calendar '${calendar.name}'.`);
                  ^


Error: Locale ID 'names:clare_of_assisi_virgin' is missing in the calendar 'Africa'.
    at <anonymous> (/home/viajante/projects/romcal-tst/rites/roman1969/build/bundle.ts:165:19)
    at Array.reduce (<anonymous>)
    at RomcalBundler (/home/viajante/projects/romcal-tst/rites/roman1969/build/bundle.ts:158:10)
    at buildPipeline (/home/viajante/projects/romcal-tst/rites/roman1969/build/build.ts:85:3)
    at <anonymous> (/home/viajante/projects/romcal-tst/rites/roman1969/build/build.ts:269:9)
    at chalk (/home/viajante/projects/romcal-tst/rites/roman1969/build/build.ts:270:1)
    at Object.<anonymous> (/home/viajante/projects/romcal-tst/rites/roman1969/build/build.ts:270:4)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Object.transformer (/home/viajante/projects/romcal-tst/node_modules/tsx/dist/register-D46fvsV_.cjs:3:1104)
    at Module.load (node:internal/modules/cjs/loader:1197:32)

Node.js v18.19.1
npm ERR! Lifecycle script `build` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: @internal/rite-roman1969@0.0.0 
npm ERR!   at location: /home/viajante/projects/romcal-tst/rites/roman1969 
```

This message _Error: Locale ID 'names:clare_of_assisi_virgin' is missing in the calendar 'Africa'_, suggests that the error is on Africa Calendar. 
**Africa Calendar is using a name that is not defined on locales/en.ts where ALL names should be defined to used.** 


bundle.ts code check locale En (English).
```
  if (locale.names && Object.prototype.hasOwnProperty.call(locale.names, id)) {
        update[id] = locale.names[id];
      } else **_if (locales.En.names && Object.prototype.hasOwnProperty.call(locales.En.names, id))_** {
        update[id] = locales.En.names[id];
      } else {
        throw new Error(`Locale ID 'names:${id}' is missing in the calendar '${calendar.name}'.`);
      }
```

--- a/rites/roman1969/build/bundle.ts
+++ b/rites/roman1969/build/bundle.ts
@@ -157,6 +157,11 @@ export const RomcalBundler = (): void => {
         .map((def) => def.i18nDef[0].substr(6))
         .reduce((obj: LocaleLiturgicalDayNames, id: string) => {
           const update: LocaleLiturgicalDayNames = { ...obj };
+          if (locales.En && !Object.prototype.hasOwnProperty.call(locales.En.names, id)) {
+            throw new Error(
+              `Locale ID 'names:${id}' is missing in the locale 'en'. Ensure the value is included in rites/roman1969/src/locales/en.ts.`
+            );
+          }
           if (locale.names && Object.prototype.hasOwnProperty.call(locale.names, id)) {
             update[id] = locale.names[id];
           } else if (locales.En.names && Object.prototype.hasOwnProperty.call(locales.En.names, id)) {

